.licenserc.yaml

```diff
@@ -35,11 +35,12 @@ header:
         // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
   - paths:
-      - "**/agent/**/*.go"
-      - "**/admin/**/.go"
+      - "./agent/**/*.go"
```

Comment:
This was too eager, so I made it more specific so it doesn't give false positives on matching directories outside of `./admin` and `./agent`. Compare before and after:

> 
atymchuk@pmm2-worker: $ bin/license-eye -c .licenserc.yaml header check
INFO Loading configuration from file: .licenserc.yaml 
INFO Totally checked 1483 files, valid: 450, invalid: 0, ignored: 1033, fixed: 0 
INFO Totally checked 184 files, valid: 165, invalid: 0, ignored: 19, fixed: 0
 
atymchuk@pmm2-worker: $ bin/license-eye -c .licenserc.yaml header check
INFO Loading configuration from file: .licenserc.yaml 
INFO Totally checked 1483 files, valid: 450, invalid: 0, ignored: 1033, fixed: 0 
INFO Totally checked 173 files, valid: 165, invalid: 0, ignored: 8, fixed: 0